resources/views/body-script.blade.php

```diff
@@ -1,6 +1,26 @@
 @if(\Astrogoat\Gtm\Settings\GtmSettings::isEnabled())
+
+    @php
+
+        $server_side_url = settings(Astrogoat\Gtm\Settings\GtmSettings::class, 'server_side_url');
+
+        if(blank($server_side_url)){
+
+            $server_side_url = 'https://www.googletagmanager.com';
+
+        }else{
+
+            if(! Str::startsWith($server_side_url,'https://')){
+                $server_side_url = Str::start($server_side_url,'https://');
+
+            }
+
+        }
+
+    @endphp
+
```

Comment:
Now that you don't have to worry about have the URL start with `https://` you can just do an simple ternary.

```suggestion
    @php
        $serverSideUrl = blank(settings(Astrogoat\Gtm\Settings\GtmSettings::class, 'server_side_url'))
            ? 'https://www.googletagmanager.com'
            : settings(Astrogoat\Gtm\Settings\GtmSettings::class, 'server_side_url');
    @endphp
```

I re-formatted the variable to be camelCase for consistency.

Reply:
maybe i was just trying to do too much! 😄

Reply:
Yeah, sometimes less is more :) Don't forget to also update the other script file, then we should be good here 👍